src/rngs/thread.rs

```diff
@@ -92,23 +95,36 @@ impl Default for ThreadRng {
     }
 }
 
+impl ThreadRng {
+    #[inline(always)]
+    fn rng(&mut self) -> &mut ReseedingRng<Core, OsRng> {
+        unsafe { &mut *self.rng.get() }
+    }
+
+    // Safe alternative using Rc<RefCell<..>>:
+    // #[inline(always)]
+    // fn rng(&mut self) -> RefMut<ReseedingRng<Core, OsRng>> {
+    //     self.rng.borrow_mut()
+    // }
```

Comment:
Do you intend to leave the dead code here?

Reply:
Probably not. I was considering using feature flags (see above), but I think it doesn't make sense overall.